pallets/creditcoin/src/benchmarking.rs

```diff
@@ -197,7 +197,7 @@ benchmarks! {
 		let who = authority_account::<T>(false);
 	}: _(root, who)
 
-	verify_transfer {
+	persist_transfer {
```

Comment:
Looks good but I am wondering why this wasn't caught by the existing CI jobs. 

We build and execute the benchmarks on each PR so I suppose something should have failed but it didn't. Need to figure out why first.

Reply:
FTR I think https://github.com/gluwa/creditcoin/pull/472 will solve the issue of being able to discover such problems.

Reply:
Did you figure it out? It needs to Compile -> Run -> Check (at least) due to the autogenerated source code.

Reply:
I think yes. In #472 I have compile; run benchmarks to generate updated wights; compile again